src/core/interfaces.tsx

```diff
@@ -0,0 +1,7 @@
+export interface ITodoItem {
```

Comment:
if I'm not mistaken, this is the interface of the props that come to the component, and not the interface itself. Accordingly, this interface needs to be named differently and does not need to be removed from the component file.